tools/packaging/common/istio-iptables.sh

```diff
@@ -193,7 +230,7 @@ for range in "${OUTBOUND_IP_RANGES_EXCLUDE[@]}"; do
     if isValidIP "$r"; then
```

Comment:
Simplify: if isIPv6... else ...

Reply:
Or even better: just let user to explicitly configure OUTBOUND_IP6_RANGES_EXCLUDE

Reply:
and if use by mistake puts garbage or invalid addresses? I have hard time understanding why you are so against of fairly common sanity verification, it is not in datapath so no performance hit, it runs only once on pod start up. what is the reason?

Reply:
The shell script is getting very complex and hard to maintain. 
We can do validation in galley - when the config is validated. Or when we switch iptable initialization to go - we can do more validation. 
The script is already extremely complex and risky to change - very few people understand it. 
Doing regex matches and validating IPv6 addresses in shell script is neither clean nor 
needed.